pkgs/development/compilers/elm/packages/elm-i18next-gen.nix

```
@@ -0,0 +1,26 @@
+{ lib, buildNpmPackage, fetchFromGitHub }:
+
+buildNpmPackage rec {
+  pname = "elm-i18next-gen";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "yonigibbs";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-RBBTJA3sEpI5nIqhifDZGldUnVvaGK2ZeYqgvccODQ0=";
+  };
+
+  npmDepsHash = "sha256-eXh73Wb5x/7DLJk5bcpn3EkCAj8of76DrRoCHNHdvxE=";
+
+  dontNpmBuild = true;
+  npmInstallFlags = "--omit=dev";
```

Please add a comment with the reason, e.g.
```suggestion

  # we can't download Elm during npm install
  npmInstallFlags = "--omit=dev";
```